src/graph.cc

```diff
@@ -277,9 +274,10 @@ std::vector<abm::graph::vertex_t> abm::Graph::dijkstra_vertices_ual(
 
   std::vector<abm::graph::vertex_t> route_vertices;
   if (path.size() > 0) {
-    for (auto itr = path.begin(); itr != path.end(); ++itr)
-      route_vertices.emplace_back(static_cast<abm::graph::vertex_t>(*itr));
-
+    for (auto itr = path.begin(); itr != path.end(); ++itr) {
+      if (itr != path.end())
```

Comment:
why is this `if` condition reintroduced here? This should match the same as `develop`

Reply:
I synced to develop rather than origin/develop previously